packages/react/src/LayoutExamples/KitchenSink.tsx

```diff
@@ -230,6 +230,59 @@ export function KitchenSink(props: KitchenSinkProps) {
             </FAQ.Answer>
           </FAQ.Item>
         </FAQ>
+        {/* TODO: Figure out what should stay and go here */}
+        {/* Adding a className to stack causes vertical direction */}
+        {/* The second trick might be applied to Stack by default */}
+        <div className="FeatureTrio">
+          <Stack direction="horizontal" padding="spacious" gap="spacious" style={{textAlign: 'center'}}>
+            <style>
+              {`
+			  	.FeatureTrio {
+					margin: 100px auto;
+				}
+
+				.FeatureTrio img {
+					box-shadow: var(--brand-River-visual-shadow);
+				}
+
+				.FeatureTrio > div > div {
+					flex: 1;
+				}
+
+				.FeatureTrio a > span {
+					color: #A475F9
```

Comment:
The example that we provide for feature-trio doesn't need to apply the purple link color from the screenshot, as that was just an example. Can we remove this? If we did enable customisation, it should also be done through css variables rather than direct hex override.